boards/arm/bl5340_dvk/Kconfig.defconfig

```diff
@@ -104,15 +104,12 @@ config BT_HCI_VS
 # https://www.buydisplay.com/2-8-inch-tft-touch-shield-for-arduino-w-capacitive-touch-screen-module
 if DISPLAY
 
-if KSCAN
-
 config INPUT_FT5336
 	default y
```

Comment:
this one is redundant, it's already default enabled, you may drop it as well while you are at it